website/docs/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -49,6 +49,12 @@ Once the HTTP server starts unauthenticated users will have to click the 'login
 
 ## Login via a cluster user account
 
+### Weave GitOps OSS
+
+If you followed the [installation instructions](../installation/weave-gitops.mdx) you don't need to perform any additional steps to enable this login method. You will be able to login to the dashboard using the credentials generated in the installation step.
```

Comment:
IMO this oversimplifies matters and doesn't point the user towards how they can configure manually should they want/need to - for example, if they have a requirement that all secrets are generated / managed via an external service.

Configuration docs should explain the mechanics and options available to the user. 

My suggestion would be to remove the OSS / WGE split and instead have a note like:

\`\`\`Weave GitOps OSS If you followed the [installation instructions](../installation/weave-gitops.mdx) you don't need to perform any additional steps to enable this login method. You will be able to login to the dashboard using the credentials generated in the installation step
\`\`\`

Reply:
Just discussed with @sympatheticmoose in-person: I will change this section to be a complete guide on the several ways you can configure a local admin user because there's so many different ways to install the dashboard that it's hard to provide step-by-step guidance.